webapp/src/models/jpo-conflictmonitor/config/ConfigDataClass.d.ts

```diff
@@ -0,0 +1,4 @@
+
+type ConfigDataClass = {
```

Comment:
Is this something that was forgotten? I don't see this being used.

Reply:
This whole directory is automatically-generated types, so there are a few that don't make complete sense. I removed it!